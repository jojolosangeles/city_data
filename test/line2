set -x
python ../code/main.py scripts/line2.txt > actual/line2.out
diff expected/line2.out actual/line2.out
diff -b expected/all_areas.Year.Victim_Sex.stacked_line.png actual
