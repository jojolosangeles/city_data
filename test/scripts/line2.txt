all_areas = expected/all_areas.csv
all_areas.drop Unnamed: 0, Date Occurred, Time Occurred, Area Name, Date, Weekday, Hour
all_areas.drop Victim Age, Victim Descent, Address, Cross Street
all_areas.columns
all_areas.xrow Year=2019
all_areas.xrow Victim Sex=X
all_areas.xrow Victim Sex=N
all_areas.xrow Victim Sex=H
all_areas.Year.Victim Sex => stacked