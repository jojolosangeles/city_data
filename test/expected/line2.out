Test: scripts/line2.txt
> all_areas = expected/all_areas.csv
Loaded expected/all_areas.csv into DataFrame 'all_areas', 453817 rows
> all_areas.drop Unnamed: 0, Date Occurred, Time Occurred, Area Name, Date, Weekday, Hour
Dropped ['Unnamed: 0', 'Date Occurred', 'Time Occurred', 'Area Name', 'Date', 'Weekday', 'Hour'] from DataFrame 'all_areas'
> all_areas.drop Victim Age, Victim Descent, Address, Cross Street
Dropped ['Victim Age', 'Victim Descent', 'Address', 'Cross Street'] from DataFrame 'all_areas'
> all_areas.columns
['Victim Sex' 'Year']
> all_areas.xrow Year=2019
451531 rows remaining
> all_areas.xrow Victim Sex=X
441844 rows remaining
> all_areas.xrow Victim Sex=N
441836 rows remaining
> all_areas.xrow Victim Sex=H
441704 rows remaining
> all_areas.Year.Victim Sex => stacked
Saved PNG stacked lines: ./actual/all_areas.Year.Victim_Sex.stacked_line.png
